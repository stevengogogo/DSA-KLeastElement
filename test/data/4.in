1500 5
-5165 1978 -6776 -9870 8381 -2616 2232 -7668 864 1284 8099 -2326 38 -6950 3881 3101 6835 9462 9188 -8747 4969 -8086 -1524 -1501 9524 8016 -1471 -9007 1426 -3361 2110 2728 -9668 -2108 7079 1996 9006 7980 1418 9145 7118 6193 3389 -2950 -5620 -4254 6378 6888 -963 -8507 7226 -1702 6280 6477 8038 -6146 6955 4051 7607 -6861 1281 3681 -5575 8378 -1387 -7423 -1412 -549 406 248 5111 -3793 9827 -3179 7806 -4010 -2245 3515 -2232 -1733 -6823 1617 -8786 8635 -1529 -8598 6500 373 -2640 -5022 467 -3382 3301 7169 -7538 -6142 -831 5244 -9289 -2025 -3685 -292 1007 1223 -9045 -3271 1446 1483 -8576 -5003 -2047 -6068 3002 1655 -8901 5232 317 -4001 -326 -72 9676 -215 -6563 -9146 6660 3121 -9689 -8970 6764 4369 -8176 7942 -8043 5180 8084 -7090 5414 5730 -5174 6080 -1034 -3990 1426 -5754 745 3059 5283 3114 -2747 2724 -8572 7834 1302 7642 1791 4986 6166 -3094 -5762 6381 772 -9711 4761 392 -2953 6452 -7011 -3762 -5226 -1701 1366 -1924 -5087 8040 -4219 462 -5407 -1872 9695 768 6365 2874 -9207 -1819 2952 -1206 3433 9511 8307 4473 -873 868 9103 2549 730 9772 4413 1702 -1752 8766 -7940 4916 -8339 9605 -4666 2469 -5060 -5188 -9347 -6879 1904 1270 9420 6560 -2985 6235 -7382 -7589 -8043 6594 286 9411 -4661 8619 9164 -7528 1405 -4558 -9025 7984 9210 -2592 -6997 -5187 301 -875 -1938 -3018 7366 3993 562 -5592 -4974 5099 -5131 -3484 -5716 5423 -2744 7975 3356 8174 1825 8033 7669 -612 7729 -9986 -8459 7588 1136 6697 7710 9810 -1165 5988 5266 -6493 -7517 2107 9426 5216 6083 5036 -6754 -9416 -8473 6872 5650 -1565 -2771 -2258 -2229 -3712 4410 -683 -2298 -9953 -494 -3116 3305 6828 -2212 -7325 -2866 -9325 -1968 -8681 1771 5858 9464 -2887 6110 -1420 6871 741 -9448 1494 952 5526 -5431 -5401 -9194 -4258 5935 577 -3445 -1984 5090 7527 -4523 -8314 -2880 3197 7203 -4436 4826 1698 -8753 -2810 1762 -3710 248 -1931 8992 4310 6637 -9531 -8985 -5580 2526 -9061 6300 2019 -4215 -2768 -9661 8113 3613 -8103 56 470 5914 2410 -1045 5225 6758 -1960 8601 -5938 -2986 7009 -316 7536 -6557 -3172 3182 247 5012 285 5950 6015 -2299 -664 3912 -5164 2898 -8251 1843 9114 8358 1443 8145 -6712 3935 4112 2475 6026 4170 4521 -4209 4804 3336 -1665 9069 3492 4388 991 -914 1866 -3400 1384 4773 9004 8096 -8044 -9877 -8485 7769 4723 1642 -8878 3026 -2554 -696 2336 -7085 -2449 -405 -2971 -1816 -2059 -302 2878 -1720 3624 7207 5417 -4781 1936 -4890 988 3506 8798 -7626 -8069 4269 -3108 9791 -311 -7231 -8742 3262 -2122 -5775 -7734 7921 3165 -2839 6167 9327 -8358 3233 -1756 7617 -4823 841 -6785 -4155 5843 8865 5595 1611 -7833 5450 5295 -3775 -9961 -6864 -492 -4478 4826 7310 6924 4034 -9829 -8674 6555 5391 1898 -773 7259 -1742 -6778 798 -6768 -5468 -8393 6815 -9140 -8642 2310 2227 4981 -9308 -8526 3413 3589 -3685 956 7352 -6147 -7 533 905 -4960 5863 -7152 3509 6920 -6910 -6301 8848 1498 -5590 7804 -6651 -1959 9021 7157 -5629 9883 8343 382 -6630 8584 4385 8449 8696 4032 -3676 -705 9192 5100 4100 -9316 3586 1730 -7144 4004 9137 4884 -8838 -9755 405 9508 1369 -1039 5881 -1384 -1091 -9296 -3263 6704 4996 -3731 8928 -2351 7075 -1782 9195 9790 -8857 1551 -2422 -3213 2708 4435 -1689 2121 -713 8914 3743 1732 7984 6558 5013 7124 2567 9541 9400 4762 2920 9611 4687 5390 7087 8663 3359 -1012 8555 -886 -6642 -7844 -81 5752 -3763 3666 -3559 5920 -7286 2584 -7028 -5914 -1420 -3493 -7140 -6287 9632 -1626 -3235 -1605 84 9469 8540 4331 6160 -3024 267 -6058 1580 6592 9329 7828 9823 4859 5495 42 -1617 -6896 -2410 -7152 9892 -7773 -8043 3919 8681 -1637 5583 5560 -3100 3494 -3465 -2198 -4140 7646 -2622 -2403 2928 -6247 2291 -8785 2490 5463 -6146 1943 4191 7759 -7556 6183 -3165 7785 -4392 -2615 6837 6065 7807 1869 -5147 -1871 6043 4616 4478 9385 -7777 -4752 -9257 8827 2690 6883 -6504 9567 -6023 -6712 -7240 -5644 -3942 4530 -3573 3783 4923 -1632 6145 352 -5729 7236 -4340 -3655 2983 -3818 3624 9128 7080 -3331 -5884 -4396 1654 -153 -3850 -6586 -6625 747 8366 7310 -2890 6442 -2641 -257 6615 -8297 -9961 5071 7472 744 8465 2941 -3062 -612 4394 -1868 -1843 3626 -2393 5461 785 26 6373 4860 -7240 -7061 -8061 -4761 -681 -7721 -2062 8041 9101 -488 9053 701 -2739 5507 -5378 -4171 -1468 -2415 -306 -4982 7290 -940 -58 -6230 599 -1326 4891 2420 -8890 4771 5498 498 -1008 5761 -5456 -3181 2765 -7379 -1801 4903 -5199 9777 -7716 -1534 7917 6682 4202 -2623 -5642 -7315 1930 -3311 -8622 3230 318 97 4629 -6836 -2432 6736 -8132 -9499 998 278 -8071 7646 -5940 584 661 -9841 2224 4373 9127 3823 -5191 -8793 4196 -3998 -6846 -4625 -9669 1458 1110 3098 5834 5914 -2245 -1767 -6245 -870 -1949 4790 -4760 -858 8014 -7027 -2122 9987 -7470 -3210 8448 -7483 -4846 1302 2343 6715 -9134 -1317 -6221 -5323 -2351 4003 7683 419 9107 -4517 -7352 178 -5682 -3049 -1953 -136 -2134 -3535 5600 5600 -5016 2835 6172 9007 -8392 3994 -3616 6117 -2058 -7236 8077 9805 1320 9428 -7982 -9031 4248 -3790 -2738 6849 -1864 -4213 -827 -3093 207 -7557 -5929 2900 -1967 -8441 -4572 -3166 -3477 -9679 1568 -482 8928 -919 2418 -3856 6121 6423 -8349 -4543 -4780 -4741 -4321 6597 5987 -1220 -7824 -8758 -7371 3009 9258 -7122 -1923 7751 3901 -28 8272 1146 1314 -5452 -9467 -1013 9497 2586 -3486 8659 9419 -2004 2965 5723 -7931 5737 -3812 -3229 -4817 -1357 8963 2266 5517 7396 -3097 3920 -4672 1105 -2419 5677 41 5361 2261 9463 9984 -6696 1178 9497 9612 -9964 3095 6287 8153 -2325 5892 -4845 7175 5352 -3286 -2572 -3167 -5561 -485 6768 3369 9357 -7265 3237 8355 5452 -4670 9278 -7834 -8356 -6836 4357 -7797 606 -3367 2453 -1481 5866 9041 5480 -9062 3652 -3928 4642 1621 6728 -8374 3217 -7660 38 -4778 -9649 -1246 6264 -3043 8730 -2089 -2054 6208 -1705 1303 -8289 -3713 -9403 -46 8516 1472 -1531 -4076 -7568 3971 1436 8812 8150 -7230 7588 -5372 3593 6253 -3946 -1932 -7759 8786 -6570 -7876 7317 -4213 5367 1137 6835 -8346 6980 -676 -6137 8195 4203 1505 -1397 8124 3840 1962 714 -6151 -4709 -5391 8098 8054 -9167 3593 3948 -5275 -5638 -3878 -9477 -3717 661 -2541 5715 -1667 -9779 3130 5230 5707 -4882 3706 2037 -46 4315 -3067 -2773 -5146 6517 6062 -6234 -705 -1390 7838 3391 -4255 6300 -5221 -6314 6583 1607 913 6221 5232 -8402 -1198 -2401 -5974 5908 5311 -1966 7027 695 5703 4169 -3619 -3588 -8715 3312 -6711 -75 5315 6749 2824 -6420 2189 3547 -4680 -1430 -9717 -2684 -9532 -8889 2786 7865 -7032 -442 -5706 7272 5444 641 653 -144 -952 -670 2107 -9515 7327 6375 3873 3818 9970 -7479 -8633 5248 6220 8108 -7433 7019 8080 -7433 -5185 7475 6419 5227 9123 3389 337 8332 -1583 -1517 -2242 -1419 6746 -661 -8416 9570 2551 -3183 -4517 -7290 5080 -4894 8044 3596 -6298 -2033 -3603 5211 -7483 -6825 -8932 8147 2816 -4655 -4211 5846 -1923 -7116 7318 2727 4939 3120 -2240 -7522 -7697 -6306 -9605 9480 -8697 4909 -7670 3720 -3587 -9169 -4018 4639 -6209 -5210 -9307 -1956 1534 -4913 -7230 -8641 9878 2270 -9105 -7086 3643 914 -3378 -1881 2300 -72 -6027 -5289 8301 4202 1869 -5552 -9000 1332 4563 2389 5731 -6140 3276 7272 -8772 -5545 867 8918 -5070 -9097 789 7266 7978 -1178 8153 8288 6092 -9266 5680 -6123 5592 -1255 4432 -7824 -487 344 1797 3292 -2168 -9983 -3914 2421 9169 -2420 2155 -753 -880 -8969 -8634 4139 4528 9286 5686 -8724 -2949 -1642 -2064 -8953 -6413 5625 4493 7135 5552 -2229 6227 -4462 -7410 -4921 9592 2283 6981 5682 -8970 8699 -3785 2265 -6870 9251 7335 8661 -5453 4671 -9781 -1689 -7445 5857 -3385 -4000 -7417 -4830 3162 2809 8535 -1377 -195 4094 -3127 -1755 1251 5384 -7529 -7649 4366 -1856 2555 1974 -2453 -4753 4900 7087 -1898 6786 -5322 595 4563 -8818 7032 3013 6652 -6935 1764 5723 -592 6843 -4316 -8356 8557 -6423 513 9069 388 816 -5464 -2664 572 -4999 -2739 -8864 4437 81 -7175 -8975 -5051 9197 -3566 -5422 8388 -9397 7464 -9651 -5534 1078 675 4551 9899 449 7107 3832 -2231 3964 5772 7766 763 -6741 -9923 3143 4974 7400 8238 615 7708 -954 4291 -5872 7089 4437 -91 4905 5222 -7851 -5779 -8403 -2900 4363 -5887 1759 -7448 1280 2609 -2006 -4043 2040 6823 -8328 6582 -18 -3666 4551 -753 3251 855 6002 7650 8167 1281 -2182 6122 -3978 -1239 9960 9967 -521 5962 9980 6887 9783 -3640 -8423 -3300 -2877 5847 8313 -4999 -2992 -2092 2391 7609 -555 180 3364 -707 -3865 5 -127 -1996 6795 5142 -1054 -318 -49 -3381 -5837 9300 3706 -1411 -7197 -2428 -6822 -651 1680 -8213 6626 -7275 -3591 6525 2645 -6940 9112 -7727 5413 7523 2936 -2184 1543
Reverse 180 1019
Delete 1314
Reverse 771 1273
Query 1290 1378 68
Query 281 907 367
