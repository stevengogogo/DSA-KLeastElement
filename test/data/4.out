4900
1898
